state-sync/diem-data-client/src/diemnet/tests.rs

```diff
@@ -162,7 +161,9 @@ async fn test_request_works_only_when_data_available() {
 
     // advance time so the poller sends a data summary request
     tokio::task::yield_now().await;
-    mock_time.advance_async(DATA_SUMMARY_POLL_INTERVAL).await;
+    mock_time
+        .advance_async(DiemDataClientConfig::default().data_summary_poll_interval)
```

Comment:
Random comment: You don't necessarily need to pass in the default timeout here. Given that it's a test, you could just hard code a value, e.g., 1000. But this works too. 😄

Reply:
Changed to use hardcoded value instead. (I think this is slightly better since it'd make it faster to change to other values if we want to.)